src/code/templates/readOutputXls.m

```diff
@@ -0,0 +1,90 @@
+function OutputList = readOutputXls(outputXls,outputsheet)
+%READOUTPUTXLS reads the output csv and translate it to an array of structures
+% see GETDEFAULTOUTPUT
+%
+% OutputList = readOutputXls(outputcsv)
+%
+% Inputs:
+%   outputXls (string) name of xls file
+%   outputsheet (string) name of sheet in slxs file, if empty first sheet is taken
+% Outputs
+%   OutputList (array of structure)  see GETDEFAULTOUTPUT
+
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+
+
+if ~exist('outputsheet','var') || isempty(outputsheet)
+    outputsheet = 1;
+end
+
+% read data
+[~,~,xlsdata] = xlsread(outputXls,outputsheet);
```

Comment:
We should call xlsread in 'basic' mode. (then we will directly see if it will work under Linux)